services/src/unit-test/java/uk/gov/moj/sdt/services/BulkFeedbackServiceTest.java

```diff
@@ -248,4 +226,30 @@ private void addValidIndividualRequest(final IBulkSubmission bulkSubmission, fin
         bulkSubmission.addIndividualRequest(individualRequest);
     }
 
+    @Test
+    void testSetBulkSubmissionDao() {
+
+        IBulkFeedbackService bulkFeedbackService = new BulkFeedbackService(mockBulkSubmissionDao, mockGlobalParameterCache);
+        IBulkSubmissionDao bulkSubmissionDaoMock = mock(BulkSubmissionDao.class);
+        ((BulkFeedbackService) bulkFeedbackService).setBulkSubmissionDao(bulkSubmissionDaoMock);
+
+        Object result = this.getAccessibleField(BulkFeedbackService.class, "bulkSubmissionDao",
+                                                IBulkSubmissionDao.class, bulkFeedbackService);
+
+        assertEquals(bulkSubmissionDaoMock, result, DAO_SHOULD_BE_SET_TO_OBJECT);
+    }
+
+    @Test
+    void testSetGlobalParametersCache() {
+
+        IBulkFeedbackService bulkFeedbackService = new BulkFeedbackService(mockBulkSubmissionDao, mockGlobalParameterCache);
+        ICacheable globalParameterCacheMock = mock(ICacheable.class);
+        ((BulkFeedbackService) bulkFeedbackService).setGlobalParametersCache(globalParameterCacheMock);
+
+        Object result = this.getAccessibleField(BulkFeedbackService.class, "globalParametersCache",
+                                                ICacheable.class, bulkFeedbackService);
+
+        assertEquals(globalParameterCacheMock, result, CACHE_OBJECT_SHOULD_BE_SET);
+    }
```

Comment:
The same comments for the testSetBulkSubmissionDao() test apply here as well.  The test can be simplified to:
```
ICacheable globalParameterCacheMock = mock(ICacheable.class);
bulkFeedbackService.setGlobalParametersCache(globalParameterCacheMock);

ICacheable result = (ICacheable) getAccessibleField(BulkFeedbackService.class,
                                                    "globalParametersCache",
                                                    ICacheable.class,
                                                    bulkFeedbackService);

assertEquals(globalParameterCacheMock, result, CACHE_OBJECT_SHOULD_BE_SET);
```

Reply:
done